src/ui-elements/Button.js

```diff
@@ -3,6 +3,7 @@
  * @class Button
  */
 export class Button {
+  // TODO: Shouldn't there be a way in this method to add a class?
```

Comment:
Was this supposed to be part of the full screen plugin? It should be included in that PR if so (also the TODO can be removed since it was actually implemented ;) )